packages/plugin-seo/src/fields/MetaDescription/MetaDescriptionComponent.tsx

```diff
@@ -58,9 +58,16 @@ export const MetaDescriptionComponent: React.FC<MetaDescriptionProps> = (props)
 
     const genDescriptionResponse = await fetch('/api/plugin-seo/generate-description', {
       body: JSON.stringify({
-        ...docInfo,
+        id: docInfo.id,
+        slug: docInfo.slug,
         doc: { ...getData() },
```

Comment:
Can you get rid of that ... spread here too? So just doc: `getData(),`

Reply:
Yep 👍 missed that one